crates/ark/src/interface.rs

```diff
@@ -142,20 +144,23 @@ pub enum SessionMode {
 // These values must be global in order for them to be accessible from R
 // callbacks, which do not have a facility for passing or returning context.
 
-// We use the `once_cell` crate for init synchronisation because the stdlib
-// equivalent `std::sync::Once` does not have a `wait()` method.
-
 /// Used to wait for complete R startup in `RMain::wait_initialized()` or
 /// check for it in `RMain::is_initialized()`.
+///
+/// We use the `once_cell` crate for init synchronisation because the stdlib
+/// equivalent `std::sync::Once` does not have a `wait()` method.
 static R_INIT: once_cell::sync::OnceCell<()> = once_cell::sync::OnceCell::new();
```

Comment:
Looks like `wait()` is a nightly method that may get stabilized soon ish https://doc.rust-lang.org/std/sync/struct.OnceLock.html#method.wait